### ISSUE TITLE
1.9.2 chunk loading issue

### ISSUE THREAD

**Issue by polkovnikov-ph** [PrismarineJS/mineflayer#434]
```
assert.js:90
  throw new assert.AssertionError({
  ^
 AssertionError: 61696 === 31589
    at addColumn (C:\Users\user\Desktop\mineflayer\lib\plugins\blocks.js:85:1
2)
    at Client.<anonymous> (C:\Users\user\Desktop\mineflayer\lib\plugins\block
s.js:188:5)
    at emitTwo (events.js:106:13)
    at Client.emit (events.js:191:7)
    at Parser.<anonymous> (C:\Users\user\Desktop\mineflayer\node_modules\mine
craft-protocol\dist\client.js:101:16)
    at emitOne (events.js:96:13)
    at Parser.emit (events.js:188:7)
    at readableAddChunk (C:\Users\user\Desktop\mineflayer\node_modules\readab
le-stream\lib\_stream_readable.js:210:18)
    at Parser.Readable.push (C:\Users\user\Desktop\mineflayer\node_modules\re
adable-stream\lib\_stream_readable.js:169:10)
    at Parser.Transform.push (C:\Users\user\Desktop\mineflayer\node_modules\r
eadable-stream\lib\_stream_transform.js:123:32)
```

**Comment by polkovnikov-ph** [comment]
I see four first fields of [this](http://wiki.vg/SMP_Map_Format) format at offset 0.

**Comment by rom1504** [comment]
Mineflayer currently only support 1.8

**Comment by polkovnikov-ph** [comment]
@rom1504 Oh, so I'll have to fix it :/

I didn't find other way than to create a new type in a protocol description and to call `bot._client.deserializer.proto.parsePacketBuffer('chunkSection', args.data)`. Am I doing it right?

**Comment by rom1504** [comment]
See https://github.com/PrismarineJS/prismarine-chunk/issues/19#issuecomment-228476338
The plan is to update prismarine-chunk for 1.9 and to use it in mineflayer.
It's not quite done yet though, if you can help on that it would be nice !

**Comment by rom1504** [comment]
closed in favor of #449
